cluster/mesos/cluster_test.go

```diff
@@ -66,11 +66,6 @@ func TestContainerLookup(t *testing.T) {
 	// Container engine/name matching.
 	assert.NotNil(t, c.Container("test-engine/container1-name1"))
 	assert.NotNil(t, c.Container("test-engine/container1-name2"))
-	// Swarm ID lookup.
-	assert.NotNil(t, c.Container("swarm1-id"))
-	// Swarm ID prefix lookup.
-	assert.NotNil(t, c.Container("swarm1-"))
-	assert.Nil(t, c.Container("swarm"))
 	// Match name before ID prefix
 	cc := c.Container("con")
 	assert.NotNil(t, cc)
```

Comment:
Not sure if testing of `com.docker.swarm.mesos.name` is also required?
I saw only a test listing `com.docker.swarm.mesos.task` here.